theano/gof/graph.py

```diff
@@ -115,6 +116,13 @@ def __init__(self, op, inputs, outputs):
             else:
                 raise TypeError("The 'outputs' argument to Apply must contain Variable instances with no owner, not %s" % output)
 
+    def run_context(self):
+        """Returns the context for the node, or NoContext if no context is set.
+        """
+        if hasattr(self.op, 'get_context'):
+            return self.op.get_context(self)
```

Comment:
It doesn't mean anything special, it's just one possible value.  The meaning of the context is Node-dependant and is not is any way enforced by this.